src/app/core/public/api/impl/Circuit.ts

```diff
@@ -196,13 +196,24 @@ export abstract class CircuitImpl implements Circuit {
     public abstract connectWire(p1: Port, p2: Port): Wire | undefined;
 
     public deleteObjs(objs: Obj[]): void {
```

Comment:
Can you also add a TSDoc-style comment to the function in `app/core/public/api/Circuit.ts` (not here in the impl)

See [this](https://github.com/OpenCircuits/OpenCircuits/blob/e3b08befc80ab8a108c1034cf670299af6f17ad2/src/app/core/public/api/Circuit.ts#L57) for an example